### PR TITLE
Centralize Translation Functions for Common Keys - Part 1 

### PR DESCRIPTION
**Pull request by pranshugupta54**
NOTE: This PR has 100+ files because we can not make changes to translations one by one - making changes to functions or components would break all other linked files. This is the base PR - further fixes and changes will be smaller.
--
> [!NOTE]
> Unauthorized test fail -> changed compare_translation file for new folder structure

**What kind of change does this PR introduce?**

Refactoring

**Issue Number:**

Reference: #1998 

**Did you add tests for your changes?**

image

**Snapshots/Videos:**

NA

**If relevant, did you update the documentation?**

NA

@coderabbitai ignore

### CODE REVIEW

**Comment by pranshugupta54** [comment]
@coderabbitai review src/components folder

**Comment by pranshugupta54** [comment]
@coderabbitai review src/screens folder

**Comment by pranshugupta54** [comment]
@coderabbitai review changes in src/components and src/screens

File: .github/workflows/compare_translations.py
Comment (pranshugupta54) [inline .github/workflows/compare_translations.py:1]: @coderabbitai review changes in this file

**Comment by pranshugupta54** [comment]
Code coverage is not directly related to this PR and can be taken separately

**Comment by palisadoes** [comment]
Please get the code coverage of the patch higher

**Comment by pranshugupta54** [comment]
Coverage is higher now. 

`src/components/OrgPostCard/OrgPostCard.tsx` - this has very less coverage and can be taken separately

**Review by palisadoes — APPROVED** [review]

**Pull request merged**